### PR TITLE
Remove legacy 'pd' attribute from course scripts

### PR DESCRIPTION
**Pull request by joshlory**
Replaced with `professional_learning_course`.

### CODE REVIEW

File: dashboard/app/views/levels/show.html.haml [hunk 1]
```diff
@@ -48,8 +48,5 @@
 
 = render partial: 'levels/admin'
 
--#
-  HACK: The respond_to? test below is to workaround a bug where a server
```
Comment (mehalshah) [inline dashboard/app/views/levels/show.html.haml:52]: Is the hack no longer needed?

Comment (joshlory) [reply dashboard/app/views/levels/show.html.haml:52]: I'm not sure why the original hack was needed.  Either way, `professional_learning_course?` is a column on `Script` so it should be present if `Script` exists.  Maybe there's some weird edge case when Dashboard is reloading that shows up in HoneyBadger, but I'm not sure we want to defend against this everywhere.

**Comment by joshlory** [comment]
Got sign-off from the PLC team that this UI change is fine to make for existing PD courses.

**Comment by mehalshah** [comment]
Cool - ship it. My change will go in separately, no need to wait

**Pull request merged**